test_weapon_magical_modifier fails

player_wizard = None, browser = <selenium.webdriver.safari.webdriver.WebDriver (session="CD5B7CB6-7249-4FC3-A736-DA9A7A022F68")>

    def test_weapon_magical_modifier(player_wizard, browser): # noqa
        """As a player, if weapon is magical, a badge indicating the modifier
           is present."""
        print(('As a player, if weapon is magical, a badge indicating the '
               'modifier is present.'))

        weapon_add = weapon.WeaponAddModal(browser)
        weapon_table = weapon.WeaponTable(browser)
        tabs = Tabs(browser)
        tabs.equipment.click()

        weapon_table.add.click()
        weapon_add.name = 'Add Name'
        weapon_add.magical_modifier = 3

        WebDriverWait(browser, 10).until(
            EC.element_to_be_clickable(
                (By.ID, weapon_table.add_id)
            )
        )

        weapon_add.add.click()

        WebDriverWait(browser, 10).until(
            modal_finished_closing(weapon_add.modal_div_id)
        )

        row = ut.get_table_row(weapon_table, 'table', 1)

>       assert row.weapon.strip() == 'Add Name  + 3'
E       AssertionError: assert 'Add Name\xa0...          + 3' == 'Add Name  + 3'
E         + Add Name  + 3
E         - Add Name
E         -               + 3

--- a/tests/core/player_tools/test_equipment.py
+++ b/tests/core/player_tools/test_equipment.py
@@ -401,7 +401,9 @@ def test_weapon_magical_modifier(player_wizard, browser): # noqa
 
     row = ut.get_table_row(weapon_table, 'table', 1)
 
-    assert row.weapon.strip() == 'Add Name  + 3'
+    weapon_name_label = ' '.join([string.strip() for string in row.weapon.split()])
+
+    assert weapon_name_label == 'Add Name + 3'
 
 
 def test_weapon_persists(player_wizard, browser): # noqa

